opdracht3.js

```diff
@@ -0,0 +1,72 @@
+// Opdracht 3a: Gebruik een array-methode om alle tv merken
+//(zoals Philips, NIKKEI, etc.) in een lijst op de pagina weer te geven.
+// Zorg ervoor dat dit ook zou werken als we 200 tv's in onze array zouden hebben staan.
+// Dat er dubbele namen in zitten, is niet erg.
+//
+let listElement = document.createElement('li');
+const id = document.getElementById('opdracht3');
+
+
+const tVBrands = inventory.map((tv) => {
+    return tv.brand;
+});
+
+let opd = [];
+let k = 0;
+let l = 0;
+for (let i = 0; i < tVBrands.length ; i++) {
+    for (let j = i + 1; j < tVBrands.length ; j++) {
+        if (tVBrands[j] === tVBrands[i]) {
+            k++;
+        }
+    }
+    if ( k === 0) {
+        opd[l] = tVBrands[i];
+        l++;
+    }
+    k = 0;
+}
```

Comment:
Het duurde even voordat ik doorhad wat je hier probeert te doen. Probeer de volgende keer dus ook wat duidelijkere variabelnamen te benoemen.

Verder zou je ditzelfde kunnen verkrijgen met de reduce:
```suggestion
const opd = tvBrands.reduce((result, brand) => {
    if (!result.includes(brand)){
        result.push(brand);
    }
    
    return result;
}, []);
}
```